generators/server/templates/dotnetcore/src/Project.Domain/Entities/AuditedEntityBase.cs.ejs

```diff
@@ -17,7 +17,7 @@ using <%= namespace %>.Domain.Entities.Interfaces;
 
 namespace <%= namespace %>.Domain.Entities;
 
-public abstract class AuditedEntityBase : IAuditedEntityBase
+public abstract class AuditedEntityBase : BaseEntity
```

Comment:
You need to do something like this

public abstract class AuditedEntityBase<TKey : BaseEntity<TKey>, IAuditedEntityBase

Reply:
> like this sorry i have write the previous code from my phone:
> 
> ```
> public abstract class AuditedEntityBase<TKey> : BaseEntity<TKey>, IAuditedEntityBase
> ```

sorry I'm new to this, Thanks for the help